src/assignment/helpers/delete-assignment.ts

```diff
@@ -64,3 +81,27 @@ export function deleteAssignmentWarning(assignment?: Avo.Assignment.Assignment):
 		'assignment/views/assignment-overview___deze-actie-kan-niet-ongedaan-gemaakt-worden'
 	);
 }
+
+export async function removeContributorFromAssignment(
+	assignmentId: string,
```

Comment:
There already is a function made for this in the assignment service called deleteContributor